src/Arcus.Templates.ServiceBus.Topic/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.0.101-alpine3.9 AS base
```

Comment:
Let's include all security patches
```suggestion
FROM mcr.microsoft.com/dotnet/core/sdk:3.0-alpine3.10 AS base
```

Reply:
Normally the base image is ASP.Net Core to provide everything related to the runtime but given this is a worker that's overkill.

However, I think we should use `runtime` rather than `sdk`.

Reply:
Not sure if this even works, or does it?

Reply:
Well, all the tests run on the Service Bus resources; so I would say, yes.